frontend/awx/views/jobs/JobOutput/JobStatusBar.tsx

```diff
@@ -60,9 +73,26 @@ export function JobStatusBar(props: { job: Job }) {
       </SplitItem>
       <SplitItem>
         <Flex>
-          <Count label={t('Plays')} count={playCount} />
-          <Count label={t('Tasks')} count={taskCount} />
-          <Count label={t('Hosts')} count={totalHostCount} />
+          <Tooltip content={t('Workflow visualizer')}>
+            <Button
```

Comment:
It might be worth considering swapping the Tooltip and Button components in favor of the framework's `PageActionButton` component. The `PageActionButton` component has tooltip, button, and spacing built-in.

Reply:
good idea :+1: done :)